region/maridia/outer.json

```diff
@@ -2240,6 +2240,40 @@
                     }}
                   ],
                   "note": "Requires a runway of approximately 12 tiles in the adjacent room."
+                },
+                {
+                  "name": "G-Mode Morph IBJ",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "h_canArtificialMorphIBJ",
+                    "Gravity"
+                  ],
+                  "note": "Kill the first fish with bombs then dodge or kill the second one."
```

Comment:
Killing fish 2, to me, would be with a power bomb.  But being able to IBJ up and retry if the fish in a bad spot is fine here, unless you are really slow with your IBJ rhythm, and that shouldn't be the case in a GMode strat.